invenio_access/cli.py

```diff
@@ -80,7 +80,7 @@ def process_role(ctx, value):
 
 option_argument = click.option(
     '-a', '--argument', default=None, metavar='VALUE',
-    help='Value for parameterized action.'
+    help='Value for parametrized action.'
```

Comment:
Both spellings is apparently [correct](https://www.merriam-webster.com/dictionary/parameterize) so I prefer keeping the old one, as we have [class names](https://github.com/inveniosoftware/invenio-access/blob/master/invenio_access/permissions.py#L41) using the same spelling :-)